Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -153,11 +191,30 @@ namespace gdjs {
     }
 
     doStepPreEvents(runtimeScene) {
+      
+      const object = this.owner;
+      
+      /* Check if the object has moved
+       * To avoid to loop on the transform and its inverse
+       * beacause of float approximation.
+       */
```

Comment:
nitpicking: use // for inline and multilines comments inside functions, as it's more concise.